main.py

```diff
@@ -142,6 +143,29 @@ def search(args, contacts: AddressBook):
     return contacts.search(args[0].strip())
 
 
+def add_note(args, notes: Notebook):
+    text = ' '.join(args)
+    return notes.create_note(text)
+
+
+def update_note(args, notes: Notebook):
+    *text, id = args
```

Comment:
Wouldn't it be more intuitive to have id first?

Reply:
I will send new pull request, but first need apply this :)

Reply:
Done